tests/test_herbstclient.py

```diff
@@ -147,6 +148,22 @@ def test_lastarg_only(hlwm, repeat):
     assert proc.returncode == 0
 
 
+@pytest.mark.parametrize('zero_separated', [True, False])  # use more repetitions to test the race-condtion
```

Comment:
Remove the "use more repetitions" comment as it doesn't apply?

Reply:
Right, thanks! It's fixed now.

Reply:
Thanks!